api/src/main/java/org/pmiops/workbench/interactiveanalysis/InteractiveAnalysisService.java

```diff
@@ -74,6 +79,7 @@ public InteractiveAnalysisService(
   public String localize(
       String workspaceNamespace,
       String appName,
+      @Nullable AppType appType,
```

Comment:
sad to add a null here, but we don't currently have a nice structure for [app type or GCE]